organizations/organizations.shacl.ttl

```diff
@@ -503,8 +528,11 @@ haOrg:OrganizationalUnitShape a sh:NodeShape ;
     a sh:PropertyShape ;
     sh:path schema:addressCountry ;
     sh:maxCount 1 ;
-    sh:datatype rdf:langString ;
-  
+    sh:or (
```

Comment:
Op zich geen probleem, maar in onze lokale semantiek (die strikter mag zijn dan Schema.org en gecapteerd is door deze SHACL) zijn landen voorlopig geen resources en dus ook geen `schema:Country`. Tenzij we een lijst van landen gaan bijhouden, maar dat is op langere termijn denk ik.

Reply:
@mielvds. Ik liet me hier leiden door de `sh:message` van de propertyshape, zie [544-546](https://github.com/viaacode/datamodels/pull/22/files#diff-60a540bd180cca9b8d6c11fcac7ae19d1a4678372c0997ad9066eec493cee74aR544-R546). 

- [ ] Ik kan het ook terugzetten naar alleen `rdf:langString`, en pas het message aan.
- [ ] Of ik laat het zo staan als dat het nu is.

Reply:
Het is ok zo!